### PR TITLE
Feature/bi 10685 create sic code api skeleton

### PR DESCRIPTION
**Pull request by JoshFen95**
__Short description outlining key changes/additions__
Creates skeleton app for the sic-code-api. The app is ran via the start.sh script which boots the app and connects it to mongodb
__JIRA Ticket Number__
10685
### Type of change

* [ ] Bug fix
* [x] New feature
* [ ] Breaking change

### Pull request checklist

* [ ] I have added unit tests for new code that I have added
* [ ] I have added/updated functional tests where appropriate
* [x] The code follows our [coding standards](https://github.com/companieshouse/styleguides/blob/master/java.md)

__An exhaustive list of peer review checks can be read [here](https://github.com/companieshouse/styleguides/blob/master/java_review.md#developer-actions-prior-to-code-commitreview-started)__

### CODE REVIEW

File: src/test/java/uk/gov/companieshouse/siccode/siccodeapi/SicCodeApiApplicationTests.java [hunk 1]
```diff
@@ -0,0 +1,13 @@
+package uk.gov.companieshouse.siccode.siccodeapi;
+
+import org.junit.jupiter.api.Test;
+import org.springframework.boot.test.context.SpringBootTest;
+
+@SpringBootTest
+class SicCodeApiApplicationTests {
+
+	@Test
```
Comment (ch-tstrickland) [inline src/test/java/uk/gov/companieshouse/siccode/siccodeapi/SicCodeApiApplicationTests.java:9]: Your IDE is set up wrong. There are tabs in the code.

File: src/main/java/uk/gov/companieshouse/siccode/siccodeapi/SicCodeApiApplication.java [hunk 2]
```diff
@@ -0,0 +1,13 @@
+package uk.gov.companieshouse.siccode.siccodeapi;
+
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+
+@SpringBootApplication
+public class SicCodeApiApplication {
+
+	public static void main(String[] args) {
```
Comment (ch-tstrickland) [inline src/main/java/uk/gov/companieshouse/siccode/siccodeapi/SicCodeApiApplication.java:9]: tabs -> spaces

File: pom.xml [hunk 3]
```diff
@@ -0,0 +1,163 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<modelVersion>4.0.0</modelVersion>
```
Comment (ch-tstrickland) [inline pom.xml:3]: Tabs -> spaces

File: pom.xml [hunk 4]
```diff
@@ -0,0 +1,163 @@
...
+	<parent>
+		<groupId>uk.gov.companieshouse</groupId>
+		<artifactId>companies-house-parent</artifactId>
+		<version>1.3.1</version>
+		<relativePath />
```
Comment (ch-tstrickland) [inline pom.xml:8]: Maybe add a comment to relative path, along the lines:
Skip the local file search and directly search the parent POM in Maven repositories

**Review by ch-tstrickland — COMMENTED** [review]
Generally looks good. How did you choose the versions of libs in maven?

**Comment by SteveHicksCH** [comment]
> Generally looks good. How did you choose the versions of libs in maven?

Created with Spring initializer

Then updated using ocr-api as a model for CH stuff (e.g. use of CH Parent pom and Sonar)

**Pull request merged**